R/trt_effects.R

```diff
@@ -110,3 +110,43 @@ mutau_cor <- function(bg,
 
 }
 
+#' Return the hypermean of the treatments effects
+ #'
+ #' @param bg        A [baggr] model.
+ #' @param transform A transformation to apply to the result, should be an R function;
+ #'                  (this is commonly used when calling `hypermean` from other
+ #'                  plotting or printing functions)
+ #' @param interval  The confidence inerval of the hypermean passed as any value between 0 and 1. 
+ #' @param message   A logical agrument that determines whether the result is printed or not.  
+ #' @return The hypermean of the `baggr` model with the interval specified by `interval`, with 
+ #'         a default interval of 95%, as calculated by [treatment_effect].
+ #' @export
+ hypermean <- function(bg,transform=NULL,interval = 0.95,message=TRUE) {
```

Comment:
(1) why not do `message = FALSE` by default?
(2) I'd add argument `summary=TRUE` and then `treatment_effect(bg, summary=summary, ...)`; that way the default behaviour is the same as what you programmed, but in the future I can also call something like `hypersd(bg, s=F)`, seems handy

Reply:
1) No real reason, I switched to the be false by default 
2) Agreed, seems handy. Added this.